rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -2120,8 +2120,7 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 
 		if (device.path.empty() || device.vid.empty() || device.pid.empty())
 		{
-			arg->out_code = CELL_ENOTSUP;
-			break;
+			return arg->out_code = CELL_ENOTMOUNTED;
```

Comment:
This syntax is ugly. Don't return and assign something at the same time.
Also, why are you changing the return value here?

Reply:
I think VFS USB devices that don't have their path, vid, and pid set should be considered not mounted.

Reply:
Updated.

Reply:
Alright. VID/PID's not being set should no longer bothers now.
A fallback value of 0x0000 is used instead.